### PR TITLE
adds mode config to Webpack config chunks production.md

### PR DESCRIPTION
**Pull request by mehrdaad**
Adds mode to production and development chunk

_describe your changes..._
This pr adds ``mode``  to Webpack chunks added in Webpack 4  

- [x] Read and sign the [CLA][1]. PRs that haven't signed it won't be accepted.
- [x] Make sure your PR complies with the [writer's guide][2].
- [x] Review the diff carefully as sometimes this can reveal issues.
- __Remove these instructions from your PR as they are for your eyes only.__

[1]: https://cla.js.foundation/webpack/webpack.js.org
[2]: https://webpack.js.org/writers-guide/

### CODE REVIEW

File: src/content/guides/production.md [hunk 1]
```diff
@@ -96,6 +98,7 @@ __webpack.prod.js__
 + const common = require('./webpack.common.js');
 +
 + module.exports = merge(common, {
++   mode: 'production',
 +   plugins: [
```
Comment (EugeneHlushko) [inline src/content/guides/production.md]: this is not how you do it in v4, go for:

```
...,
optimization: {
  minimizer: [
    new UglifyJsPlugin({}),
    ...
  [
}
```

Comment (mehrdaad) [reply src/content/guides/production.md]: Should it be removed?

Comment (EugeneHlushko) [reply src/content/guides/production.md]: yes mode: production does minification now by default, providing a custom minimizer only lets you override default options

Comment (mehrdaad) [reply src/content/guides/production.md]: Ok, I updated it

File: src/content/guides/production.md [hunk 2]
```diff
@@ -92,17 +94,14 @@ __webpack.prod.js__
...
 + module.exports = merge(common, {
-+   plugins: [
-+     new UglifyJSPlugin()
-+   ]
++   mode: 'production',
 + });
 ```
 
-In `webpack.common.js`, we now have our `entry` and `output` setup configured and we've included any plugins that are required for both environments. In `webpack.dev.js`, we've added the recommended `devtool` for that environment (strong source mapping), as well as our simple `devServer` configuration. Finally, in `webpack.prod.js`, we included the `UglifyJSPlugin` which was first introduced by the [tree shaking](/guides/tree-shaking) guide.
+In `webpack.common.js`, we now have our `entry` and `output` setup configured and we've included any plugins that are required for both environments. In `webpack.dev.js`,we've set ``mode`` to ``development``.Also, we've added the recommended `devtool` for that environment (strong source mapping), as well as our simple `devServer` configuration. Finally, in `webpack.prod.js`,``mode`` is set to ``production`` which loads `UglifyJSPlugin` which was first introduced by the [tree shaking](/guides/tree-shaking) guide.
```
Comment (montogeek) [inline src/content/guides/production.md]: Please add a space before `we've set` and before `.Also`

**Comment by mehrdaad** [comment]
Done.

**Comment by mehrdaad** [comment]
@montogeek is this merged?

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**